content/fast/adaptive-serving-based-on-network-quality/codelab-adaptive-serving-based-on-network-quality.md

```diff
@@ -0,0 +1,172 @@
+---
+page_type: glitch
+title: Adaptive serving based on network quality
```

Comment:
Said below as well, I'd change this title to be something like "Adapt video to image serving based on network quality'.